diff --git a/dist/react.d.ts b/dist/react.d.ts
index 6c42997d24f2b03237bd9b09a0f8ab61940440fd..77023cfce80b6e0af32791d5827a53980610fa2a 100644
--- a/dist/react.d.ts
+++ b/dist/react.d.ts
@@ -13,6 +13,6 @@ export declare const useModuleConstructor: <T extends ModuleHandler>({ Handler,
         callbacks?: Record<string, GenericMessageCallback> | undefined;
         element?: HTMLElement | null | undefined;
     }, "element"> | undefined;
-    ref: RefObject<HTMLElement>;
+    ref: RefObject<HTMLElement | null>;
 }) => T;
 export {};
