diff --git a/dist/get-dependents-graph.cjs.dev.js b/dist/get-dependents-graph.cjs.dev.js
index 94dde7b0aa903cf4aa099acfd2fc62f6243264d9..21b56dee9c7e0e8f1100efe00be3e6091d62d1b3 100644
--- a/dist/get-dependents-graph.cjs.dev.js
+++ b/dist/get-dependents-graph.cjs.dev.js
@@ -71,6 +71,9 @@ function getDependencyGraph(packages, opts) {
     for (let [depName, depRange] of allDependencies) {
       const match = packagesByName[depName];
       if (!match) continue;
+      if (match.packageJson.private) {
+        continue;
+      }
       const expected = match.packageJson.version;
       const usesWorkspaceRange = depRange.startsWith("workspace:");
 
diff --git a/dist/get-dependents-graph.cjs.prod.js b/dist/get-dependents-graph.cjs.prod.js
index 39b2dfe0c15d052f4e342a81626c4012a695867b..e2d7b359022d1f422f01e04860e81538a7f16f1b 100644
--- a/dist/get-dependents-graph.cjs.prod.js
+++ b/dist/get-dependents-graph.cjs.prod.js
@@ -45,6 +45,9 @@ function getDependencyGraph(packages, opts) {
     for (let [depName, depRange] of allDependencies) {
       const match = packagesByName[depName];
       if (!match) continue;
+      if (match.packageJson.private) {
+        continue;
+      }
       const expected = match.packageJson.version;
       if (depRange.startsWith("workspace:")) {
         if (depRange = depRange.replace(/^workspace:/, ""), "*" === depRange || "^" === depRange || "~" === depRange) {
diff --git a/dist/get-dependents-graph.esm.js b/dist/get-dependents-graph.esm.js
index 5f29582339d3a6a45ff882caf843ecf8a9d6c99d..c969d70e7fad6b65e7dcf222d122b1fd8d341d81 100644
--- a/dist/get-dependents-graph.esm.js
+++ b/dist/get-dependents-graph.esm.js
@@ -62,6 +62,9 @@ function getDependencyGraph(packages, opts) {
     for (let [depName, depRange] of allDependencies) {
       const match = packagesByName[depName];
       if (!match) continue;
+      if (match.packageJson.private) {
+        continue;
+      }
       const expected = match.packageJson.version;
       const usesWorkspaceRange = depRange.startsWith("workspace:");
 
