diff --git a/dist-es/utils/shared-props.d.ts b/dist-es/utils/shared-props.d.ts
index ba8f0355357d748d0f359fdbe87db5395412e50d..f1062dffbfeef0e4798232459c4206e559367ec5 100755
--- a/dist-es/utils/shared-props.d.ts
+++ b/dist-es/utils/shared-props.d.ts
@@ -63,6 +63,7 @@ export interface PickerProps {
     /** NOTE: default is not preventable */
     onClick?(emoji: EmojiData, e: React.MouseEvent<HTMLElement>): void;
     onSelect?(emoji: EmojiData): void;
+    onEmojiSelect?(emoji: EmojiData): void;
     onSkinChange?(skin: EmojiSkin): void;
     perLine?: number | undefined;
     emojiSize?: number | undefined;
