diff --git a/node_modules/@blockprotocol/graph/dist/es/stdlib-temporal.js b/node_modules/@blockprotocol/graph/dist/es/stdlib-temporal.js
index b911b36..75dbae2 100644
--- a/node_modules/@blockprotocol/graph/dist/es/stdlib-temporal.js
+++ b/node_modules/@blockprotocol/graph/dist/es/stdlib-temporal.js
@@ -1660,7 +1660,7 @@ const getOutgoingLinksForEntity$1 = (subgraph, entityId, interval) => {
     const entities = [];
     for (const [edgeTimestamp, outwardEdges] of typedEntries(entityEdges)) {
         // Only look at outgoing edges that were created before or within the search interval
-        if (!intervalIsStrictlyAfterInterval$1(intervalForTimestamp$1(edgeTimestamp), searchInterval)) {
+        if (intervalIntersectionWithInterval$1(searchInterval, intervalForTimestamp(edgeTimestamp))) {
             for (const outwardEdge of outwardEdges) {
                 if (isTemporalSubgraph(subgraph)) {
                     const outwardEdgeTemporal = outwardEdge;
