diff --git a/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs b/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs
index 32ccb00..ff5347d 100644
--- a/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs
+++ b/node_modules/@blockprotocol/graph/dist/cjs/codegen.cjs
@@ -4,7 +4,7 @@ var path = require('node:path');
 var slim = require('@blockprotocol/type-system/slim');
 var jsonSchemaToTypescript = require('json-schema-to-typescript');
 var fs = require('node:fs/promises');
-var fetch = require('node-fetch');
+// PATCH: rely on native fetch until https://github.com/blockprotocol/blockprotocol/pull/1310 is merged and released
 var ts = require('typescript');
 
 function _interopNamespaceDefault(e) {
