diff --git a/node_modules/@blockprotocol/graph/dist/types.d.ts b/node_modules/@blockprotocol/graph/dist/types.d.ts
index 20797c8..b0263dd 100644
--- a/node_modules/@blockprotocol/graph/dist/types.d.ts
+++ b/node_modules/@blockprotocol/graph/dist/types.d.ts
@@ -116,6 +116,7 @@ export declare type EntityType = {
         $schema: string;
         title: string;
         type: string;
+        labelProperty: string;
         [key: string]: unknown;
     };
 };
