diff --git a/node_modules/@types/prosemirror-view/index.d.ts b/node_modules/@types/prosemirror-view/index.d.ts
index 4d2e67d..b6b2ccf 100755
--- a/node_modules/@types/prosemirror-view/index.d.ts
+++ b/node_modules/@types/prosemirror-view/index.d.ts
@@ -594,19 +594,16 @@ export interface EditorProps<ThisT = unknown, S extends Schema = any> {
      * the node view without adding it to the document itself.
      */
     nodeViews?:
+        // This patch is related to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57384
+        // These errors were observed before patching: https://github.com/hashintel/hash/pull/206#issuecomment-1018826336
         | {
-              [name: string]: ((
+              [name: string]: (
                   node: ProsemirrorNode<S>,
                   view: EditorView<S>,
                   /** get the node's current position */
                   getPos: () => number,
                   decorations: Decoration[],
-              ) => NodeView<S>) | ((
-                  mark: Mark<S>,
-                  view: EditorView<S>,
-                  /** indicates whether the mark's content is inline */
-                  inline: boolean,
-              ) => Pick<NodeView<S>, 'dom' | 'contentDOM'>);
+              ) => NodeView<S>;
           }
         | null
         | undefined;
