diff --git a/node_modules/blockprotocol/core.d.ts b/node_modules/blockprotocol/core.d.ts
index 205630a..b78cb42 100644
--- a/node_modules/blockprotocol/core.d.ts
+++ b/node_modules/blockprotocol/core.d.ts
@@ -66,6 +66,8 @@ export type BlockProtocolMultiSort = {
 }[];
 
 export type BlockProtocolAggregateOperationInput = {
+  entityTypeId?: string;
+  entityTypeVersionId?: string | null;
   pageNumber?: number;
   itemsPerPage?: number;
   multiSort?: BlockProtocolMultiSort | null;
@@ -74,13 +76,10 @@ export type BlockProtocolAggregateOperationInput = {
 
 export type BlockProtocolLinkedDataDefinition = {
   aggregate?: BlockProtocolAggregateOperationInput & { pageCount?: number };
-  entityTypeId?: string;
   entityId?: string;
 };
 
 export type BlockProtocolAggregateEntitiesPayload = {
-  entityTypeId?: string;
-  entityTypeVersionId?: string | null;
   operation: BlockProtocolAggregateOperationInput;
   accountId?: string;
 };
@@ -139,6 +138,7 @@ export type BlockProtocolEntity = {
 };
 
 export type BlockProtocolLink = {
+  id: string;
   sourceEntityId: string;
   destinationEntityId: string;
   destinationEntityVersionId?: string | null;
@@ -170,8 +170,7 @@ export type BlockProtocolCreateLinksFunction = {
 export type BlockProtocolDeleteLinksAction = {
   sourceAccountId?: string | null;
   sourceEntityId: string;
-  index?: number | null;
-  path: string;
+  linkId: string;
 };
 
 export type BlockProtocolDeleteLinksFunction = {
