diff --git a/node_modules/blockprotocol/core.d.ts b/node_modules/blockprotocol/core.d.ts
index 205630a..6ab9516 100644
--- a/node_modules/blockprotocol/core.d.ts
+++ b/node_modules/blockprotocol/core.d.ts
@@ -66,6 +66,7 @@ export type BlockProtocolMultiSort = {
 }[];
 
 export type BlockProtocolAggregateOperationInput = {
+  entityTypeId?: string;
   pageNumber?: number;
   itemsPerPage?: number;
   multiSort?: BlockProtocolMultiSort | null;
@@ -91,6 +92,7 @@ export type BlockProtocolAggregateEntitiesResult<T = unknown> = {
 };
 
 export type BlockProtocolAggregateEntityTypesPayload = {
+  accountId?: string;
   includeOtherTypesInUse: boolean;
 };
 
@@ -112,10 +114,12 @@ export type BlockProtocolFileMediaType = "image" | "video";
 
 export type BlockProtocolUploadFileFunction = {
   (action: {
+    accountId?: string;
     file?: File;
     url?: string;
     mediaType: BlockProtocolFileMediaType;
   }): Promise<{
+    accountId?: string | null;
     entityId: string;
     url: string;
     mediaType: BlockProtocolFileMediaType;
@@ -139,6 +143,7 @@ export type BlockProtocolEntity = {
 };
 
 export type BlockProtocolLink = {
+  linkId: string;
   sourceEntityId: string;
   destinationEntityId: string;
   destinationEntityVersionId?: string | null;
@@ -153,25 +158,60 @@ export type BlockProtocolLinkGroup = {
   links: BlockProtocolLink[];
 };
 
+export type BlockProtocolLinkedAggregationOperationInput = BlockProtocolAggregateOperationInput & {
+  entityTypeId: string;
+};
+
+export type BlockProtocolUpdateLinksMutationResults = {
+  sourceAccountId: string;
+  sourceEntityId: string;
+  path: string;
+  operation: BlockProtocolLinkedAggregationOperationInput & {
+    pageCount?: number;
+  };
+}
+
+export type BlockProtocolLinkedAggregation = BlockProtocolUpdateLinksMutationResults & {
+  results: Array<BlockProtocolEntity>;
+  pageCount?: number;
+  entityTypeId: string;
+}
+
 export type BlockProtocolCreateLinksAction = {
   sourceAccountId?: string | null;
   sourceEntityId: string;
+  sourceEntityTypeId?: string;
+  sourceEntityTypeVersionId?: string;
+  index?: number | null;
+  path: string;
+} & { 
+  operation?: BlockProtocolAggregateOperationInput; 
+} | {  
   destinationAccountId?: string | null;
   destinationEntityId: string;
   destinationEntityVersionId?: string | null; 
-  index?: number | null;
-  path: string;
 };
 
 export type BlockProtocolCreateLinksFunction = {
   (actions: BlockProtocolCreateLinksAction[]): Promise<BlockProtocolLink[]>;
 };
 
+export type BlockProtocolUpdateLinksAction = {
+  sourceAccountId?: string;
+  sourceEntityId: string;
+  path: string;
+  data: BlockProtocolLinkedAggregationOperationInput;
+};
+
+export type BlockProtocolUpdateLinksFunction = {
+  (actions:BlockProtocolUpdateLinksAction[]): Promise<BlockProtocolUpdateLinksMutationResults[]>;
+};
+
 export type BlockProtocolDeleteLinksAction = {
+  linkId: string;
   sourceAccountId?: string | null;
   sourceEntityId: string;
   index?: number | null;
-  path: string;
 };
 
 export type BlockProtocolDeleteLinksFunction = {
@@ -185,6 +225,7 @@ export type BlockProtocolAggregateEntityTypesFunction = {
 };
 
 type BlockProtocolUpdateEntityTypesAction = {
+  accountId?: string;
   entityId: string;
   schema: JSONObject;
 };
@@ -219,6 +260,7 @@ export interface JSONArray extends Array<JSONValue> {}
  * which the embedding application should provide.
  */
 export type BlockProtocolProps = {
+  accountId?: string;
   aggregateEntities?: BlockProtocolAggregateEntitiesFunction;
   aggregateEntitiesLoading?: boolean;
   aggregateEntitiesError?: Error;
@@ -234,8 +276,10 @@ export type BlockProtocolProps = {
   deleteLinksError?: Error;
   entityId?: string;
   entityTypeId?: string;
+  entityTypeVersionId?: string;
   linkedEntities?: BlockProtocolEntity[];
   linkGroups?: BlockProtocolLinkGroup[];
+  linkedAggregations?: BlockProtocolLinked[];
   id?: string;
   schemas?: Record<string, JSONObject>;
   type?: string;
@@ -245,4 +289,6 @@ export type BlockProtocolProps = {
   updateEntityTypes?: BlockProtocolUpdateEntityTypesFunction;
   updateEntityTypesLoading?: boolean;
   updateEntityTypesError?: Error;
+  updateLinks?: BlockProtocolUpdateLinksFunction;
+  uploadFile?: BlockProtocolUploadFileFunction;
 };
