diff --git a/node_modules/blockprotocol/core.d.ts b/node_modules/blockprotocol/core.d.ts
index 205630a..cf61c16 100644
--- a/node_modules/blockprotocol/core.d.ts
+++ b/node_modules/blockprotocol/core.d.ts
@@ -63,24 +63,17 @@ export type BlockProtocolMultiFilter = {
 export type BlockProtocolMultiSort = {
   field: string;
   desc?: boolean | undefined | null;
+  __typename?: string;
 }[];
 
-export type BlockProtocolAggregateOperationInput = {
-  pageNumber?: number;
-  itemsPerPage?: number;
-  multiSort?: BlockProtocolMultiSort | null;
-  multiFilter?: BlockProtocolMultiFilter | null;
-};
+
 
 export type BlockProtocolLinkedDataDefinition = {
   aggregate?: BlockProtocolAggregateOperationInput & { pageCount?: number };
-  entityTypeId?: string;
   entityId?: string;
 };
 
 export type BlockProtocolAggregateEntitiesPayload = {
-  entityTypeId?: string;
-  entityTypeVersionId?: string | null;
   operation: BlockProtocolAggregateOperationInput;
   accountId?: string;
 };
@@ -116,6 +109,7 @@ export type BlockProtocolUploadFileFunction = {
     url?: string;
     mediaType: BlockProtocolFileMediaType;
   }): Promise<{
+    accountId: string;
     entityId: string;
     url: string;
     mediaType: BlockProtocolFileMediaType;
@@ -139,6 +133,7 @@ export type BlockProtocolEntity = {
 };
 
 export type BlockProtocolLink = {
+  linkId: string;
   sourceEntityId: string;
   destinationEntityId: string;
   destinationEntityVersionId?: string | null;
@@ -153,6 +148,40 @@ export type BlockProtocolLinkGroup = {
   links: BlockProtocolLink[];
 };
 
+export type BlockProtocolAggregateOperationInput = {
+  entityTypeVersionId?: string | null;
+  pageNumber?: number;
+  itemsPerPage?: number;
+  multiSort?: BlockProtocolMultiSort | null;
+  multiFilter?: BlockProtocolMultiFilter | null;
+  entityTypeId?: string;
+};
+
+export type BlockProtocolAggregateOperationResults = BlockProtocolAggregateOperationInput & {
+  pageCount?: number;
+};
+
+export type BlockProtocolLinkedAggregationOperationInput = BlockProtocolAggregateOperationInput & {
+  entityTypeId: string;
+};
+
+export type BlockProtocolUpdateLinksMutationResults = {
+  sourceAccountId: string;
+  sourceEntityId: string;
+  path: string;
+  operation: BlockProtocolLinkedAggregationOperationInput & {
+    pageCount?: number;
+  };
+}
+
+export type BlockProtocolLinkedAggregation = BlockProtocolUpdateLinksMutationResults & {
+  __typename?: string;
+  results: Array<BlockProtocolEntity>;
+  pageCount?: number;
+  entityTypeId: string;
+}
+
+
 export type BlockProtocolCreateLinksAction = {
   sourceAccountId?: string | null;
   sourceEntityId: string;
@@ -167,11 +196,21 @@ export type BlockProtocolCreateLinksFunction = {
   (actions: BlockProtocolCreateLinksAction[]): Promise<BlockProtocolLink[]>;
 };
 
+export type BlockProtocolUpdateLinksAction = {
+  sourceAccountId: string;
+  sourceEntityId: string;
+  path: string;
+  updatedOperation: BlockProtocolLinkedAggregationOperationInput;
+};
+
+export type BlockProtocolUpdateLinksFunction = {
+  (actions:BlockProtocolUpdateLinksAction[]): Promise<BlockProtocolUpdateLinksMutationResults[]>;
+};
+
 export type BlockProtocolDeleteLinksAction = {
   sourceAccountId?: string | null;
   sourceEntityId: string;
-  index?: number | null;
-  path: string;
+  linkId: string;
 };
 
 export type BlockProtocolDeleteLinksFunction = {
@@ -219,6 +258,7 @@ export interface JSONArray extends Array<JSONValue> {}
  * which the embedding application should provide.
  */
 export type BlockProtocolProps = {
+  accountId?: string;
   aggregateEntities?: BlockProtocolAggregateEntitiesFunction;
   aggregateEntitiesLoading?: boolean;
   aggregateEntitiesError?: Error;
@@ -236,6 +276,7 @@ export type BlockProtocolProps = {
   entityTypeId?: string;
   linkedEntities?: BlockProtocolEntity[];
   linkGroups?: BlockProtocolLinkGroup[];
+  linkedAggregations?: BlockProtocolLinked[];
   id?: string;
   schemas?: Record<string, JSONObject>;
   type?: string;
@@ -245,4 +286,6 @@ export type BlockProtocolProps = {
   updateEntityTypes?: BlockProtocolUpdateEntityTypesFunction;
   updateEntityTypesLoading?: boolean;
   updateEntityTypesError?: Error;
+  updateLinks?: BlockProtocolUpdateLinksFunction;
+  uploadFile?: BlockProtocolUploadFileFunction;
 };
