diff --git a/node_modules/blockprotocol/core.d.ts b/node_modules/blockprotocol/core.d.ts
index 205630a..1cf433a 100644
--- a/node_modules/blockprotocol/core.d.ts
+++ b/node_modules/blockprotocol/core.d.ts
@@ -66,6 +66,8 @@ export type BlockProtocolMultiSort = {
 }[];
 
 export type BlockProtocolAggregateOperationInput = {
+  entityTypeId?: string;
+  entityTypeVersionId?: string | null;
   pageNumber?: number;
   itemsPerPage?: number;
   multiSort?: BlockProtocolMultiSort | null;
@@ -74,13 +76,10 @@ export type BlockProtocolAggregateOperationInput = {
 
 export type BlockProtocolLinkedDataDefinition = {
   aggregate?: BlockProtocolAggregateOperationInput & { pageCount?: number };
-  entityTypeId?: string;
   entityId?: string;
 };
 
 export type BlockProtocolAggregateEntitiesPayload = {
-  entityTypeId?: string;
-  entityTypeVersionId?: string | null;
   operation: BlockProtocolAggregateOperationInput;
   accountId?: string;
 };
@@ -116,6 +115,7 @@ export type BlockProtocolUploadFileFunction = {
     url?: string;
     mediaType: BlockProtocolFileMediaType;
   }): Promise<{
+    accountId: string;
     entityId: string;
     url: string;
     mediaType: BlockProtocolFileMediaType;
@@ -139,6 +139,7 @@ export type BlockProtocolEntity = {
 };
 
 export type BlockProtocolLink = {
+  linkId: string;
   sourceEntityId: string;
   destinationEntityId: string;
   destinationEntityVersionId?: string | null;
@@ -153,6 +154,42 @@ export type BlockProtocolLinkGroup = {
   links: BlockProtocolLink[];
 };
 
+export type BlockProtocolAggregateOperation = {
+  entityTypeId?: string;
+  entityTypeVersionId?: string | null;
+  multiFilter?: {
+    filters: {
+      field: string;
+      value: string;
+      operator: BlockProtocolFilterOperatorType;
+    }[]
+    operator: BlockProtocolMultiFilterOperatorType;
+  } | null;
+  multiSort?: {
+    field: string;
+    desc?: boolean | null;
+  }[] | null;
+  itemsPerPage?: number;
+  pageNumber?: number;
+  pageCount?: number;
+};
+
+export type BlockProtocolLinkedAggregationUpdated = {
+  sourceAccountId: string;
+  sourceEntityId: string;
+  path: string;
+  operation: BlockProtocolAggregateOperation;
+}
+
+export type BlockProtocolLinkedAggregation = {
+  sourceAccountId: string;
+  sourceEntityId: string;
+  path: string;
+  operation: BlockProtocolAggregateOperation;
+  results: Array<BlockProtocolEntity>;
+}
+
+
 export type BlockProtocolCreateLinksAction = {
   sourceAccountId?: string | null;
   sourceEntityId: string;
@@ -167,11 +204,21 @@ export type BlockProtocolCreateLinksFunction = {
   (actions: BlockProtocolCreateLinksAction[]): Promise<BlockProtocolLink[]>;
 };
 
+export type BlockProtocolUpdateLinkedAggregationsAction = {
+  sourceAccountId: string;
+  sourceEntityId: string;
+  path: string;
+  updatedOperation: AggregateOperationInput;
+};
+
+export type BlockProtocolUpdateLinkedAggregationsFunction = {
+  (actions:BlockProtocolUpdateLinkedAggregationsAction[]): Promise<BlockProtocolLinkedAggregationUpdated[]>;
+};
+
 export type BlockProtocolDeleteLinksAction = {
   sourceAccountId?: string | null;
   sourceEntityId: string;
-  index?: number | null;
-  path: string;
+  linkId: string;
 };
 
 export type BlockProtocolDeleteLinksFunction = {
@@ -219,6 +266,7 @@ export interface JSONArray extends Array<JSONValue> {}
  * which the embedding application should provide.
  */
 export type BlockProtocolProps = {
+  accountId?: string;
   aggregateEntities?: BlockProtocolAggregateEntitiesFunction;
   aggregateEntitiesLoading?: boolean;
   aggregateEntitiesError?: Error;
@@ -236,6 +284,7 @@ export type BlockProtocolProps = {
   entityTypeId?: string;
   linkedEntities?: BlockProtocolEntity[];
   linkGroups?: BlockProtocolLinkGroup[];
+  linkedAggregations?: BlockProtocolLinked[];
   id?: string;
   schemas?: Record<string, JSONObject>;
   type?: string;
@@ -245,4 +294,6 @@ export type BlockProtocolProps = {
   updateEntityTypes?: BlockProtocolUpdateEntityTypesFunction;
   updateEntityTypesLoading?: boolean;
   updateEntityTypesError?: Error;
+  updateLinkedAggregations?: BlockProtocolUpdateLinkedAggregationsFunction;
+  uploadFile?: BlockProtocolUploadFileFunction;
 };
