error[graph-read-compiler::nested-graph-read-unsupported]: Nested graph read operations not supported
   ╭▸ 
19 │ ┌       ["==",
20 │ │         "read",
21 │ │         //~^ ERROR Nested graph read operations not supported here
22 │ │         "read"
   │ │          ━━━━ Nested graph read operations not supported here
23 │ │         //~^ ERROR Nested graph read operations not supported here
24 │ │       ]
   │ └───────┘ ... within this filter expression
   │
   ├ help: Filter expressions do not support nested graph operations. Use a separate query for the nested graph read and pass the result to this filter expression.
   ╰ note: Nested graph reads in filter expressions are not yet implemented. This is a current limitation that is being tracked in https://linear.app/hash/issue/H-4913/hashql-implement-vm.

error[graph-read-compiler::nested-graph-read-unsupported]: Nested graph read operations not supported
   ╭▸ 
19 │ ┌       ["==",
20 │ │         "read",
   │ │          ━━━━ Nested graph read operations not supported here
21 │ │         //~^ ERROR Nested graph read operations not supported here
22 │ │         "read"
23 │ │         //~^ ERROR Nested graph read operations not supported here
24 │ │       ]
   │ └───────┘ ... within this filter expression
   │
   ├ help: Filter expressions do not support nested graph operations. Use a separate query for the nested graph read and pass the result to this filter expression.
   ╰ note: Nested graph reads in filter expressions are not yet implemented. This is a current limitation that is being tracked in https://linear.app/hash/issue/H-4913/hashql-implement-vm.