error[graph-read-compiler::nested-graph-read-unsupported]: Nested graph read operations not supported
   ╭▸ 
19 │ ┌       ["==",
20 │ │         "read",
21 │ │         //~^ ERROR Nested graph operation not supported here
22 │ │         "read"
   │ │          ━━━━ Nested graph operation not supported here
23 │ │         //~^ ERROR Nested graph operation not supported here
24 │ │       ]
   │ └───────┘ ... within this filter expression
   │
   ├ help: Filter expressions do not currently support nested graph operations. Move the graph operation outside the filter expression, assign the result to a variable, and use that variable in the filter instead.
   ╰ note: Nested graph operations in filter expressions are not yet implemented. This is a current limitation that is being tracked in https://linear.app/hash/issue/H-4913/hashql-implement-vm and https://linear.app/hash/issue/H-4915/hashql-hoist-nested-graph-operations-inside-filters.

error[graph-read-compiler::nested-graph-read-unsupported]: Nested graph read operations not supported
   ╭▸ 
19 │ ┌       ["==",
20 │ │         "read",
   │ │          ━━━━ Nested graph operation not supported here
21 │ │         //~^ ERROR Nested graph operation not supported here
22 │ │         "read"
23 │ │         //~^ ERROR Nested graph operation not supported here
24 │ │       ]
   │ └───────┘ ... within this filter expression
   │
   ├ help: Filter expressions do not currently support nested graph operations. Move the graph operation outside the filter expression, assign the result to a variable, and use that variable in the filter instead.
   ╰ note: Nested graph operations in filter expressions are not yet implemented. This is a current limitation that is being tracked in https://linear.app/hash/issue/H-4913/hashql-implement-vm and https://linear.app/hash/issue/H-4915/hashql-hoist-nested-graph-operations-inside-filters.